README.md

```diff
@@ -28,19 +28,21 @@ By [PostHog](https://posthog.com).
 
 All inputs are **optional**. If not set, sensible defaults will be used.
 
-| Name   | Description             | Default |
-| ------ | ----------------------- | ------- |
-| `path` | Path to the npm package | `.`     |
+| Name                  | Description                                            | Default |
+| --------------------- | ------------------------------------------------------ | ------- |
+| `path`                | Path to the npm package                                | `.`     |
+| `allow-first-version` | Do not raise an error when package is not found on npm | `false` |
 
 ### Action outputs
 
 The following outputs can be used by subsequent workflow steps.
 
-| Name                | Description                                                                               |
-| ------------------- | ----------------------------------------------------------------------------------------- |
-| `committed-version` | Version now commited to the repo                                                          |
-| `published-version` | Latest version published to npm, based on `dist-tags`                                     |
-| `is-new-version`    | Whether repo version is new to npm (has not been published before), `'true'` or `'false'` |
+| Name                | Description                                                                                        |
+| ------------------- | -------------------------------------------------------------------------------------------------- |
+| `committed-version` | Version now committed to the repo                                                                  |
+| `published-version` | Latest version published to npm, based on `dist-tags` (this won't be set for the first version)    |
+| `is-new-version`    | Whether repo version is newer than npm version (or package does not exists), `'true'` or `'false'` |
+| `is-first-version`  | Whether repo package does not exist on npm, `'true'` or `'false'`                                  |
```

Comment:
`is-first-version` is not correct, because it'd be false for a package which _is_ published, except with only one version in npm which is the same as locally. That'd be a first version too.
Seems like this should be more specifically `is-new-package`